packages/python/plotly/plotly/express/_core.py

```diff
@@ -1106,7 +1164,7 @@ def process_args_into_dataframe(args, wide_mode, var_name, value_name):
     reference.
     """
 
-    df_input = args["data_frame"]
+    df_input: nw.DataFrame | None = args["data_frame"]
```

Comment:
I am adding type hints in multiple places for easier developing experience. However the codebase has no type annotation elsewhere. Please let me know if it's ok to keep them or if I should remove them